cypress/README.md

```diff
@@ -25,7 +40,7 @@ Storybook must be running before Cypress tests can be run:
 1. Run Storybook `npm start`.
 2. Open a new terminal in the root path of the project.
 3. Run Cypress using the runner with `npx cypress open` or `npm run test-cypress`, then select the required feature file. Test results can be seen directly in the Cypress Test Runner UI.
-4. To run specific Cypress tests at the command line (headless browser for continuous integration) use: `npx cypress run --spec 'cypress/features/[tests-type]/[featureFileName].feature'`. Test results can be seen in the console run summary.
+4. To run specific Cypress tests at the command line (headless browser for continuous integration) use: `npx cypress run --spec 'cypress/integration/[tests-type]/[featureFileName].feature'|.test.js`. Test results can be seen in the console run summary.
```

Comment:
```suggestion
4. To run specific Cypress tests at the command line (headless browser for continuous integration) use: `npx cypress run --spec 'cypress/integration/[tests-type]/[featureFileName](.feature|.test.js)'`. Test results can be seen in the console run summary.
```

Might not be exactly the right syntax but should be something like that I think

Reply:
it is not a syntax just to show what we can run :)

Reply:
updated